### PR TITLE
[systeminfo] Update oshi to 3.4.5 release

### PR DESCRIPTION
**Pull request by falkena**
Simply update systeminfo binding oshi dependency to latest release.

Jar to test can be found under
https://openhab.jfrog.io/openhab/libs-pullrequest-local/org/openhab/binding/org.openhab.binding.systeminfo/2.3.0-SNAPSHOT/org.openhab.binding.systeminfo-2.3.0-SNAPSHOT.jar

Signed-off-by: Alexander Falkenstern <alexander.falkenstern@gmail.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.systeminfo/.classpath [hunk 1]
```diff
@@ -4,9 +4,9 @@
     path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8" />
   <classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins" />
   <classpathentry kind="src" path="src/main/java" />
-  <classpathentry exported="true" kind="lib" path="lib/jna-platform-4.4.0.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/jna-4.4.0.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/oshi-core-3.4.3.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/threetenbp-1.3.5.jar" />
+  <classpathentry exported="true" kind="lib" path="lib/jna-platform-4.5.0.jar" />
+  <classpathentry exported="true" kind="lib" path="lib/jna-4.5.0.jar" />
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.systeminfo/.classpath]: According to the [changelog](https://github.com/oshi/oshi/blob/master/CHANGELOG.md#350-41518) OSHI 3.5.0 depends on JNA 4.5.1.

Comment (falkena) [reply addons/binding/org.openhab.binding.systeminfo/.classpath]: Thank you very much for review. According to openHab log, at least on Raspberry, JNA 4.5.1 will not work properly: 
grafik

Downgrade to JNA 4.5.0 worked. There is a similar error, i think, but not sure about, for oshi 3.4.4, is reported here: https://github.com/oshi/oshi/issues/463

**Comment by cweitkamp** [comment]
Has OSHI 3.5.0 introduced any important changes compared to its predecessor 3.4.5? If not I suggest to update the binding with OSHI 3.4.5 instead to fulfill the libraries requirements and make sure we will not introduce side effects in the binding.

**Comment by martinvw** [comment]
Thanks guys!

**Pull request merged**